### DISCUSSION TITLE
status icons in custom css

### DISCUSSION THREAD

**Discussion by pinkjimrussiagirl** [stoatchat/discussions#620] [Q&A] [answered]
i am trying to add the discord status icons as the status icons for revolt in my custom css, but i do not know how to do it or if there is a way to do it without having to use javascript

here is a screenshot of what the status icons look like that i am trying to add
![IMG_6903](https://github.com/revoltchat/revolt/assets/111091825/4f6e98be-5275-4659-87d9-c233e880ddeb)

here is a screenshot of where i am trying to add the discord style status icons
![IMG_6904](https://github.com/revoltchat/revolt/assets/111091825/efecd7fc-3216-471e-b11e-2c23121c03a8)

when i select the status icons in inspect element it comes up with an element called “<circle>” and idk how i should change the shapes of it using only the custom css

note: i had to remake the discussion because no one commented on the older one

**Comment by Rexogamer** [comment, marked as answer]
Closing as a duplicate of #379. Please be patient.
